tools/aa/commands/WorkspaceCommand.java

```diff
@@ -0,0 +1,55 @@
+package com.google.startupos.tools.aa.commands;
+
+import com.google.startupos.common.FileUtils;
+import com.google.startupos.common.flags.Flag;
+import com.google.startupos.common.flags.FlagDesc;
+import com.google.startupos.common.flags.Flags;
+import java.io.IOException;
+import java.nio.file.Paths;
+import javax.inject.Inject;
+
+public class WorkspaceCommand implements AaCommand {
+  @FlagDesc(
+    name = "force",
+    description = "Force workspace switching creating workspace if non-existent"
```

Comment:
s/Force workspace switching creating workspace if non-existent/Create workspace if it doesn't exist